### ISSUE TITLE
`np.fmax(npa, 1e-5))` does not consider that npa is negative

### ISSUE THREAD

**Issue by acgtyrant** [vacancy/Synchronized-BatchNorm-PyTorch#13]
See https://github.com/vacancy/Synchronized-BatchNorm-PyTorch/blob/master/sync_batchnorm/unittest.py#L28

If npa is negative at all, then np.fmax will return `1e-5`, even the abs of npa is greater than `1e-5`.

**Comment by acgtyrant** [comment]
By the way, `np.fmax(npa, 1e-5))` is use for when npa is 0 at all?

**Comment by vacancy** [comment]
Thanks for the reporting. I think it should be `np.fmax(np.abs(npa), 1e-5).max()`. I will fix it later.
